### ISSUE TITLE
ABIDE Performance

### ISSUE THREAD

**Issue by tuttyfrutyee** [xxlya/BrainGNN_Pytorch#6]
Hi, loved your work. Are the hyperparameters in this repository optimized for ABIDE? Because when I run main.py, the train accuracy is 0.676 and test accuracy 0.57 after 99 epochs and they are not getting any better. Is there anything to do to get higher accuracy specifically for ABIDE? Thanks...

**Comment by xxlya** [comment]
We gave the example of ABIDE in this repo because its data is publicly available. However, we did not pay much effect to improve its performance. It is more like an example. Also, it may be difficult to get good performance for ABIDE as it is very noisy.. sometimes more complex model makes learning worse. The two datasets we used in our paper are more cleaner. We are trying to make our HCP preprocess pipeline public.

**Comment by htlee6** [comment]
Hey @tuttyfrutyee , not sure if you were aware of some facts about the ABIDE dataset. The images in this dataset are taken at different sites and with different devices. So the standards of imaging and a lot of other factors affect, and you can regard them as noises. So if you fed the entire dataset to the model without denoising, I think that won't lead to a good result.
